mkt/webapps/tests/test_indexers.py

```diff
@@ -263,6 +256,22 @@ def test_trending(self):
         ok_('trending_2' not in doc)
 
 
+class TestHomescreenIndexer(TestCase):
+    def test_mapping_type_name(self):
+        eq_(HomescreenIndexer.get_mapping_type_name(), 'homescreen')
+
+    def test_index(self):
+        with self.settings(ES_INDEXES={'homescreen': 'homescreens'}):
+            eq_(HomescreenIndexer.get_index(), 'homescreens')
+
+    def test_model(self):
+        eq_(HomescreenIndexer.get_model(), Webapp)
+
+    def test_mapping(self):
+        mapping = HomescreenIndexer.get_mapping()
+        eq_(mapping.keys(), ['homescreen'])
+
```

Comment:
Forgot this earlier: add a test on `get_indexable` here and in `TestWebappIndexer`, to show that these indexers properly include/ignore homescreens